apps/backoffice-v2/src/domains/workflow-definitions/fetchers.ts

```diff
@@ -14,11 +15,27 @@ export const PluginSchema = z.object({
 
 export type TPlugin = z.infer<typeof PluginSchema>;
 
+export const WorkflowDefinitionConfigTheme = z.object({
+  type: z
+    .union([
+      z.literal(WorkflowDefinitionConfigThemeEnum.KYB),
+      z.literal(WorkflowDefinitionConfigThemeEnum.KYC),
+      z.literal(WorkflowDefinitionConfigThemeEnum.DOCUMENTS_REVIEW),
+    ])
```

Comment:
There is no longer a need for `z.literal`.
```suggestion
    .enum(WorkflowDefinitionConfigThemeEnum)
```

Reply:
It wont work like that for key:value enum. It expects array of static values to correctly project type.

Reply:
See how we did it in other places.